modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/events/ClientExtensionsServicePreAction.java

```diff
@@ -117,6 +112,36 @@ private CET _getCET(
 			companyId, clientExtensionEntryRel.getCETExternalReferenceCode());
 	}
 
+	private ThemeCSSCET _getControlPanelThemeCSSCET(Layout layout) {
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-34650")) {
+			return null;
+		}
```

Comment:
It's a good practice for `get` methods  only return state, without control flow. What do you think about move out this feature flag check ?